arch/arc/core/thread.c

```diff
@@ -115,7 +115,11 @@ void _new_thread(struct k_thread *thread, k_thread_stack_t *stack,
 #endif
 
 #ifdef CONFIG_ARC_HAS_SECURE
+#ifdef CONFIG_ARC_SECURE_FIRMWARE
```

Comment:
Is reading the current contents best way to initialize the SEC_STAT in the initial context? Can't it be some constant value?

Reply:
It could be  a constan value.  I will consider to change it later.